test/utility/misc.c

```diff
@@ -601,6 +603,34 @@ testutil_time_us(WT_SESSION *session)
     return ((uint64_t)ts.tv_sec * WT_MILLION + (uint64_t)ts.tv_nsec / WT_THOUSAND);
 }
 
+/*
+ * testutil_pareto --
+ *     Given a random value generated by __wt_random, a key range and a skew percentage generate a
+ *     pareto distributed value.
+ */
+uint32_t
+testutil_pareto(uint32_t rand, uint64_t min_key, uint64_t max_key, u_int skew)
+{
+    double S1, S2, U;
```

Comment:
I'm not a huge fan of these variable names but I don't know what they stand for either.